starter-code/index.html

```diff
@@ -35,7 +35,7 @@
         <header>
           <h1>Title</h1>
           <div class="byline">
-            By <address><a href="">Author Name</a></address>
+            By <address><a class="url" href="">Author Name</a></address>
```

Comment:
Is this class needed? Can't we just target the class in the div and target the a tag below that class? We want to do as little editing of our starter code unless it is asked for!